lib/registries/addon/components/registries-states/is-public/component.ts

```diff
@@ -0,0 +1,16 @@
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import styles from './styles';
+import layout from './template';
+
+export default class RegistrationWithdrawal extends Component {
+    styles = styles;
+    layout = layout;
+
+    @service i18n!: I18N;
+
+    scientistName?: string = 'Curie';
```

Comment:
Looks like this is the only component that need that have `scientistName`?

Reply:
Right. only this component should handle withdrawal form.